backend/src/main/java/com/rota/api/StaffService.java

```diff
@@ -80,4 +81,17 @@ public boolean hasManagerPermissions(String authString) {
         .filter(staff -> !staff.isInactive())
         .collect(Collectors.toList());
   }
+
```

Comment:
Should we have a differentiation between deactivate and remove? Removing a staff member for me sounds like a permanent removal, and this is just deactivating. How would a manager permanently remove a member of staff, or is that not something they would ever do?

Reply:
I thought the idea was that remove=deactivate in our case, so we get to keep all the user's past engagements intact.
Correct me if I'm wrong.